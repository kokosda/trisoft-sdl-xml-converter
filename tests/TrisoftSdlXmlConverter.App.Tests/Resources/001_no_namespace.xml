<?xml version="1.0" encoding="utf-8"?>
<html>
    <head title="@!Trisoft!!@ makes perfect!">SDL is so-o-o good!</head>
    <body title="Body should not contain title | Trisoft">
        <h1>Trisoft will be replaced soon by SDL Trisoft</h1>
    </body>
</html>