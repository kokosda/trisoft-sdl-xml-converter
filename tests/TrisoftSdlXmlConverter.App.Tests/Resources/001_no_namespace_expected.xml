<?xml version="1.0" encoding="utf-8"?>
<html xmlns:html4="http://www.w3.org/TR/html4">
    <head title="@!SDL Trisoft!!@ makes perfect!">SDL is so-o-o good!</head>
    <body title="Body should not contain title | SDL Trisoft">
        <h1>SDL Trisoft will be replaced soon by SDL Trisoft</h1>
    </body>
</html>